capellambse/model/common/accessors.py

```diff
@@ -275,9 +285,10 @@ def _make_list(self, parent_obj, elements):
 class DirectProxyAccessor(WritableAccessor[T], PhysicalAccessor[T]):
     """Creates proxy objects on the fly."""
 
-    __slots__ = ("follow_abstract", "rootelem")
+    __slots__ = ("follow_abstract", "rootelem", "single_attr")
```

Comment:
The `single_attr` slot would actually belong to `WritableAccessor`, I'm not sure how much sense it makes to have it defined in all its subclasses. I'd rather not.

Reply:
I tried to have `single_attr` on the `WritableAccessor` where it should belong to... but Python doesn't let me because of 
`TypeError: multiple bases have instance lay-out conflict`.

Have a look at [this great answer for usage of `__slots__` and its caveats](https://stackoverflow.com/questions/472000/usage-of-slots#answer-28059785). The interesting section for our problem here is named `Biggest Caveat: Multiple inheritance`.

Reply:
Solution: Just don't include this attribute in the slots.